add_docstrings.py

```diff
@@ -221,9 +242,38 @@ def leave_IndentedBlock(
             seen.add(updated.name.value)
             body.append(updated)
 
-        self.level -= 1
         return updated_node.with_changes(body=body)
 
+    def leave_IndentedBlock(
+        self, original_node: libcst.IndentedBlock, updated_node: libcst.IndentedBlock
+    ) -> libcst.IndentedBlock:
+        return self._leave_Suite(original_node, updated_node)
+
+    def leave_Module(
+        self, original_node: libcst.Module, updated_node: libcst.Module
+    ) -> libcst.Module:
+        return self._leave_Suite(original_node, updated_node)
+
+    def leave_If(self, original_node: libcst.If, updated_node: libcst.If) -> libcst.If:
+        condition_as_libcst_module = libcst.Module(
+            body=[
+                libcst.SimpleStatementLine(body=[libcst.Expr(value=original_node.test)])
+            ]
+        )
+        condition_as_ast_expr = ast.parse(condition_as_libcst_module.code).body[0]
+        assert isinstance(condition_as_ast_expr, ast.Expr)
+        literal_eval_visitor = typeshed_client.parser._LiteralEvalVisitor(
+            self.typeshed_client_context, self.stub_file_path
+        )
```

Comment:
@JelleZijlstra -- don't suppose I could persuade you to consider making this public, could I...?